utils/helpers.py

```diff
@@ -0,0 +1,2 @@
+def percentage_from(number: float, percent: int) -> float:
+    return (number * percent) / 100
```

Comment:
As this function works with money, better make it work with Decimal instead of float.

Reply:
Resolved